packages/transition-backend/src/services/executableJob/ExecutableJob.ts

```diff
@@ -149,23 +150,48 @@ export class ExecutableJob<TData extends JobDataType> extends Job<TData> {
         });
     }
 
-    getFilePath = (fileName: keyof TData['files']): string | undefined => {
+    hasFile(file: keyof TData['files']): boolean {
+        const fileName = this.getFileName(file);
+        if (fileName === undefined) {
+            return false;
+        }
+        return true;
+    }
+
+    getFileName = (file: keyof TData['files']): string | undefined => {
```

Comment:
Might be worth documenting exactly what each of those method does. Between `hasFile`, `getFileName` and `getFilePath`, what is expected. In `getFilePath`, for example, it returns the absolute path to an existing file, while `getFileName` just returns the name of the file but does not say if it exists or not? Does `hasFile` implies its existence?